apps/judicial-system/digital-mailbox-api/src/app/app.config.ts

```diff
@@ -1,15 +1,28 @@
 import { defineConfig } from '@island.is/nest/config'
 
-export default defineConfig({
-  name: 'AppModule',
+export const authModuleConfig = defineConfig({
+  name: 'AuthModule',
```

Comment:
I think we should name this something other than AuthModule (and AppModule to be fair wasn't the best name either). Maybe just DigitalMailboxModule or something like that